tsdfileapi/api.py

```diff
@@ -2574,7 +2575,7 @@ def get(self, tenant: str, table_name: str = None) -> None:
         except Exception as e:
             logging.error(e)
             self.set_status(400)
-            self.write({'message': self.error})
+            self.write({'message': f"Internal Error: {type(e).__name__}"})
```

Comment:
400 is a Bad Request, not an Internal Error. If you want to change the message then change the exception handling to distinguish the two from each other

Reply:
Yes it is. I also introduced an error when the result is empty, so I will correct both in the following commit.